tests/integration_tests/security/api_tests.py

```diff
@@ -138,6 +138,85 @@ def test_post_guest_token_bad_resources(self):
         self.assert400(response)
 
 
+@pytest.mark.usefixtures("load_birth_names_dashboard_with_slices", scope="class")
+class TestSecurityGuestTokenApiTokenValidator(SupersetTestCase):
+    uri = "api/v1/security/guest_token/"  # noqa: F541
```

Comment:
Is this noqa needed? Afaict F541 is "don't use unnecessary f-strings", which you haven't here. The linter didn't complain when I removed it just now.